connector/tests/src/test/java/org/oaebudt/edc/ManagementApiTransferTest.java

```diff
@@ -46,13 +45,14 @@
 
 @Testcontainers
 @EndToEndTest
+@PostgresqlIntegrationTest
```

Comment:
these annotations just set "tags" that can be used to decide which test gets executed from the command line.
given that the CI workflow [currently already runs the `EndToEndTest`](https://github.com/OAEBUDT/oaebudt-dataspace/blob/87f8d1fc5e44b52b2efaeefc670c0bcc0cf47589/.github/workflows/connector-pr-validation.yml#L74) there's no need to add this annotation, unless we split the tests between in-memory and postgres (not recommended unless really needed)